projects/fsastorefrontlib/src/cms-components/change-process/change-coverage/change-coverage.component.ts

```diff
@@ -27,9 +27,9 @@ export class ChangeCoverageComponent extends AbstractChangeProcessStepComponent
             );
             if (optionalProducts && !this.isSimulated(changeRequestData)) {
               optionalProducts.map((coverage, index) => {
-                coverage.index = index;
-                this.optionalCoverages.push(coverage);
-                this.initialOptionalCoverages.push({ ...coverage });
+                const newCoverage = { ...coverage, index };
```

Comment:
newCoverage can be misleading because here we are not adding a new one to the list of optionals.
Maybe to create const optionalCoverage as copy of original coverage?

Reply:
`optionalCoverage` sounds very specific, could be maybe misunderstood for an actual value coming somewhere from BE or something like that. Would it be ok if I change const to `coverageClone` or `coverageCopy` to be very clear on its' purpose?

Reply:
Ok, change it to one of these two proposals.
And fix that merge conflict in fnol navigation component :)

Reply:
Done :)